library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,61 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+For any particular library dependency chain, the libraries involved need to
+coordinate the adoption of new major versions of any shared dependency that has
```

Comment:
Can we shorten this to "new major versions of shared dependencies."?

Reply:
Done.